pkg/iam/route53_template.go

```diff
@@ -41,6 +41,13 @@ const route53TrustIdentityPolicyWithIRSA = `{
 const route53RolePolicyTemplate = `{
```

Comment:
[optional]

Should we add a link to https://cert-manager.io/docs/configuration/acme/dns01/route53/ where the required policy seems documented?

Reply:
Thought about it but this policy isn't just for `cert-manager`